tests/test_lightningd.py

```diff
@@ -312,10 +312,10 @@ def fund_channel(self, l1, l2, amount):
         decoded2 = bitcoind.rpc.decoderawtransaction(tx)
         raise ValueError("Can't find {} payment in {} (1={} 2={})".format(amount, tx, decoded, decoded2))
 
-    def line_graph(self, n=2):
+    def line_graph(self, n=2, options=None):
```

Comment:
An improvement certainly, but not used in the same commit that contains it. In principle, better to separate in a different commit.  Fine by me either way though.

Reply:
Removed changes, I initially used `line_graph` in the onchaind test but then needed to instruct one to fail, so I used the primitives instead.